app/models/maestrano/connector/rails/concerns/complex_entity.rb

```diff
@@ -27,8 +27,8 @@ def formatted_entities_names(names)
       names.index_by { |name| name }.with_indifferent_access
     end
 
-    def count_entities(entities)
-      entities.values.map(&:size).max
+    def count_and_first(entities)
```

Comment:
Might be worth adding a comment to point out that for a complex_entity is get biggest array and count?